tests/App/AppKernel.php

```diff
@@ -102,6 +102,8 @@ protected function configureContainer(ContainerBuilder $containerBuilder, Loader
             'strict_variables' => '%kernel.debug%',
             'exception_controller' => null,
             'form_themes' => ['@SonataAdmin/Form/form_admin_fields.html.twig'],
+            'paths' => ['%kernel.project_dir%/Resources/views'],
+
```

Comment:
You can remove this extra line

Reply:
WIP. I will probably need it in functional tests. Set templates must exists.

Reply:
I was talking about the blank line

Reply:
:smile: